README.md

```diff
@@ -50,8 +50,19 @@ to provide the best of both worlds.
         1. ***void fetchAd(AdRetriever retriever, RetrieveAdCallback callback)*** 
         2. ***void fetchQuePoint(CuePointsRetriever retriever, CuePointCallBack callBack)***
         
-
-
+    5. If you want to quickly test the pre-roll and mid-roll video ads using the default components, just simply change 1 line of code in [PlayerModuleDefault](lib/src/main/java/com/tubitv/media/di/PlayerModuleDefault.java#L131): <br/>  
+    ```java
+  
+       @Override
+       public void fetchQuePoint(CuePointsRetriever retriever, CuePointCallBack callBack) {
+    
+           callBack.onCuePointReceived(new long[] { 60000, 900000, 1800000, 3600000 });
+                //"AdBreak point at 0s, 1min, 15min, 30min, 60min. With each adbreak showing one ads"
```

Comment:
You have "0s" in the comment but don't use 0 in the array.  Let's delete it fir consistency.  Also, "one ad" singular.

Reply:
good catch, i just added 0 in the new commit